mlflow/spark.py

```diff
@@ -614,7 +614,7 @@ def _load_model_databricks(model_uri, dfs_tmpdir):
     # Copy the model to a temp DFS location first. We cannot delete this file, as
     # Spark may read from it at any point.
     fuse_dfs_tmpdir = dbfs_hdfs_uri_to_fuse_path(dfs_tmpdir)
-    os.mkdir(fuse_dfs_tmpdir)
+    os.makedirs(fuse_dfs_tmpdir)
```

Comment:
```suggestion
    os.makedirs(fuse_dfs_tmpdir, exist_ok=True)
```

Do we need `exist_ok=True`?

Reply:
`dfs_tmpdir` looks like it's a temp directory so we probably don't need this.